### ISSUE TITLE
just a question about ysm files

### ISSUE THREAD

**Issue by RedstoneGyerek** [Elaina69/Yes-Steve-Model-Repo#1]
henlo
is there any way to modify ysm files? or convert them to zip or something?

**Comment by Elaina69** [comment]
no, ysm is a encrypted file, you can NOT extract and edit it as zip model

**Comment by DamienDesmont** [comment]
Cómo se puede descodificar el archivo.ysm y extraerlos archivos @Elaina69

**Comment by DamienDesmont** [comment]
Hello?

**Comment by Elaina69** [comment]
> Cómo se puede descodificar el archivo.ysm y extraerlos archivos @Elaina69 

as i said, ysm is a encrypted file, you can NOT extract and edit it

**Comment by DamienDesmont** [comment]
Pero,el creador del archivo no sabe si el puede convertir el archivo ysm a .bbmodel?

**Comment by DamienDesmont** [comment]
Solo se que en su página de Twitter está la info

**Comment by DamienDesmont** [comment]
> también usa inglés en su lugar

Ok

**Comment by DamienDesmont** [comment]
but, how can I download or buy the 3d models with the .bbmodel extension

**Comment by DamienDesmont** [comment]
You could ask the author how to get them but not encryption

**Comment by DamienDesmont** [comment]
![file](https://github.com/Elaina69/Yes-Steve-Model-Repo/assets/155505068/a0d804a2-9ace-4fcf-ad8d-3d80cb1ec5e9)

**Comment by DamienDesmont** [comment]
Hello?

**Comment by DamienDesmont** [comment]
Hello?
